index.js

```diff
@@ -40,6 +40,12 @@ function handleOctaveMsg(msg) {
   }
 }
 
+function handleBofaMsg(msg) {
+  if (msg.content.toLowerCase() == 'bofa') {
```

Comment:
instead of checking the message content in the handler, make a regex and check in the event handler.
`if (bofaRegex.test(msg.cleanContent) { try { handleBofaMsg(msg) } }`

Reply:
Will do